tensorflow_quantum/python/layers/circuit_executors/expectation_test.py

```diff
@@ -171,6 +191,69 @@ def test_static_cases(self):
                                   symbol_values=[[0.5]],
                                   operators=test_psum)
 
+    def test_static_cases_nois(self):
```

Comment:
Need a case somewhere here where `repetitions` is 2D

Reply:
Good catch, added that case.